plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/price-filter.block_theme.spec.ts

```diff
@@ -0,0 +1,165 @@
+			page,
+			defaultBlockPost,
+		} ) => {
+			await page.goto(
+				`${ defaultBlockPost.link }?min_price=20&max_price=67`
+			);
+
+			// Min price input field
+			const leftInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-left-input'
+			);
+			const minPriceInput = leftInputContainer.locator( '.min' );
+			const minPriceInputValue = await minPriceInput.inputValue();
+
+			expect( minPriceInputValue ).toBe( '$20' );
+
+			// Min price slider thumb
+			const priceSlider = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-price-range-slider'
+			);
+			const minPriceThumb = priceSlider.locator( 'input[name="min"]' );
+			const minPriceThumbValue = await minPriceThumb.inputValue();
+
+			expect( minPriceThumbValue ).toBe( '20' );
+
+			// Max price input field
+			const rightInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-right-input'
+			);
+			const maxPriceInput = rightInputContainer.locator( '.max' );
+			const maxPriceInputValue = await maxPriceInput.inputValue();
+
+			expect( maxPriceInputValue ).toBe( '$67' );
+
+			// Max price slider thumb
+			const maxPriceThumb = priceSlider.locator( 'input[name="max"]' );
+			const maxPriceThumbValue = await maxPriceThumb.inputValue();
+
+			expect( maxPriceThumbValue ).toBe( '67' );
+		} );
+
+		test( 'Changes in the price input field triggers price slider updates', async ( {
+			page,
+			defaultBlockPost,
+		} ) => {
+			await page.goto(
+				`${ defaultBlockPost.link }?min_price=20&max_price=67`
+			);
+
+			// Min price input field
+			const leftInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-left-input'
+			);
+			const minPriceInput = leftInputContainer.locator( '.min' );
+			await minPriceInput.fill( '30' );
+			await minPriceInput.blur();
+			const minPriceInputValue = await minPriceInput.inputValue();
+
+			expect( minPriceInputValue ).toBe( '$30' );
+
+			// Min price slider thumb
+			const priceSlider = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-price-range-slider'
+			);
+			const minPriceThumb = priceSlider.locator( 'input[name="min"]' );
+			const minPriceThumbValue = await minPriceThumb.inputValue();
+
+			expect( minPriceThumbValue ).toBe( '30' );
+
+			// Max price input field
+			const rightInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-right-input'
+			);
+			const maxPriceInput = rightInputContainer.locator( '.max' );
+			await maxPriceInput.fill( '80' );
+			await maxPriceInput.blur();
+			const maxPriceInputValue = await maxPriceInput.inputValue();
+
+			expect( maxPriceInputValue ).toBe( '$80' );
+
+			// Max price slider thumb
+			const maxPriceThumb = priceSlider.locator( 'input[name="max"]' );
+			const maxPriceThumbValue = await maxPriceThumb.inputValue();
+
+			expect( maxPriceThumbValue ).toBe( '80' );
```

Comment:
It would be good for consistency to also have the `toHaveValue` assertion used here as well.

Reply:
Nice catch! Fixed in https://github.com/woocommerce/woocommerce/pull/45855/commits/f982d74acbce9e9c8a28a681a454de5b494e5c13